src/components/tables/Captures/Rows.tsx

```diff
@@ -135,6 +135,7 @@ function Row({ isSelected, setRow, row, showEntityStatus }: RowProps) {
                 detailsExpanded={detailsExpanded}
                 id={row.last_pub_id}
                 colSpan={tableColumns.length}
+                browserTitleKey="captureDetails"
```

Comment:
We should take this out in a follow up PR. I noticed the browserTitle when looking at details page and forgot that details started as a page. Now that it is a panel we don't need it.

Reply:
Follow up PR: #187